.eslintrc.json

```diff
@@ -7,6 +7,8 @@
     "jsx-a11y/label-has-for":0,
     "react/jsx-closing-tag-location":0,
     "import/no-named-as-default":0,
+    "jsx-a11y/anchor-is-valid": 0,
+    "react/forbid-prop-types": 0,
```

Comment:
Line -10: This will allow us to use react-router's `Link` without providing a `href`. In other words - eslint will not flag a `Link` tag without a `href` attribute. 

Line-11: Because we do not know the exact properties that would be in objects in an array, I think it's safe to use the array propType. So I had to disable rules forbidding it.

Reply:
cool